src/app/provider/onboarding/verify-phone/usePhoneVerification.ts

```
@@ -0,0 +1,164 @@
+                  step: "phone-invalid",
+                  error: errorMessage,
+                });
+                phoneVerificationForm.setError("phoneNumber", {
+                  type: "manual",
+                  message:
+                    errorMessage ||
+                    "Your phone number is invalid. Please try again.",
+                });
+                reject(new Error(`${errorMessage}`));
+              }
+            });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)